stanford_soe_r25.module

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * @file
+ * Code for the Stanford SOE R25 module.
+ */
+
+/**
+ * Hook into the reservation form for insidesoe.
+ */
+function stanford_soe_r25_form_stanford_r25_reservation_alter(&$form, &$form_state, $form_id) {
+  $form['stanford_r25_booking_tentative']['#weight'] = -11;
+  $form['stanford_r25_booking_date']['#weight'] = -10;
+  $form['stanford_r25_booking_duration']['#weight'] = -9;
+  $form['stanford_r25_booking_enddate']['#weight'] = -8;
+  $form['stanford_r25_booking_reason']['#weight'] = -7;
+  $form['stanford_r25_booking_attr191']['#weight'] = -3;
+  $form['stanford_r25_booking_headcount']['#weight'] = -2;
+  $form['stanford_r25_booking_attr144']['#weight'] = -1;
+  $form['stanford_r25_booking_reason']['#title'] = 'Event Title';
```

Comment:
Form title text should be translated with `t()`. Please translate all title strings.